File: HW/HW2 學生成績資料卡/Date.cpp
#include "Date.h"

Date::Date()
{
	month = 1;
	day = 1;
	year = 1;
}
Date::Date(int newMonth, int newDay, int newYear)
{
	setMonth(newMonth);
	setDay(newDay);
	setYear(newYear);
}
void Date::setMonth(int newMonth)
{
	month = newMonth;
}
void Date::setDay(int newDay)
{
	day = newDay;
}
void Date::setYear(int newYear)
{
	year = newYear;
}
int Date::getMonth()const
{
	return month;
}
int Date::getDay()const
{
	return day;
}
int Date::getYear()const
{
	return year;
}

File: 期末考2.0/Q4 Python Composition/Date.py
class Date():
    def __init__(self, newYear, newMonth, newDay):
        self.__year = newYear
        self.__month = newMonth
        self.__day = newDay
    def setYear(self, newYear):
        self.__year = newYear
    def setMonth(self, newMonth):
        self.__month = newMonth
    def setDay(self, newDay):
        self.__day = newDay
    def getYear(self):
        return self.__year
    def getMonth(self):
        return self.__month
    def getDay(self):
        return self.__day
    def toString(self):
        print("{0}-{1}-{2}".format(self.getYear(),self.getMonth(),self.getDay()))

